homeassistant/components/sensibo/quality_scale.yaml

```diff
@@ -0,0 +1,77 @@
+rules:
+  # Bronze
+  config-flow:
+    status: todo
+    comment: |
+      Data descriptions.
+  test-before-configure: done
+  unique-config-entry: done
+  config-flow-test-coverage: done
+  runtime-data: done
+  test-before-setup: done
+  appropriate-polling: done
+  entity-unique-id: done
+  has-entity-name: done
+  entity-event-setup:
+    status: exempt
+    comment: |
+      Entities doesn't subscribe to events.
+  dependency-transparency: done
+  action-setup:
+    status: exempt
+    comment: |
+      No integrations services.
+  common-modules: done
+  docs-high-level-description: todo
+  docs-installation-instructions: done
+  docs-removal-instructions: todo
+  docs-actions: done
+  brands: done
+  # Silver
+  config-entry-unloading: done
+  log-when-unavailable: done
+  entity-unavailable: done
+  action-exceptions: done
+  reauthentication-flow: done
+  parallel-updates: done
+  test-coverage: done
```

Comment:
Please patch the library to where we use them?
```py
    with patch("pysensibo.SensiboClient.async_get_devices", return_value=load_json):
        output = await client.async_get_devices_data()
```

Reply:
All tests in Sensibo needs to be rewritten to function somewhat proper (currently the tests are influencing each other etc. etc.)

Reply:
Should we note that down so people don't mistakenly use these tests as examples for their integration?

Reply:
Gong to do it in the next day as I want to update  some stuff and the tests makes it somewhat impossibiel. E.g. 5 minutes for a feature, an hour for fixing the tests...

Reply:
Understandable, but I also kinda want this PR to be merged quickly, otherwise it will be blocked by a lot of other PRs and take ages to merge